virt/virt-4-12-release-notes.adoc

```diff
@@ -179,6 +179,13 @@ Some features in this release are currently in Technology Preview. These experim
 
 link:https://access.redhat.com/support/offerings/techpreview[Technology Preview Features Support Scope]
 
+//CNV-20526
+* The Tekton Tasks Operator (TTO) now xref:../virt/virtual_machines/virt-managing-vms-openshift-pipelines.adoc#virt-managing-vms-openshift-pipelines[integrates {VirtProductName} with {pipelines-title}]. TTO includes cluster tasks and example pipelines that allow you to:
++
+** Create and manage virtual machines (VMs), persistent volume claims (PVCs), and data volumes
+** Run commands in VMs
+** Manipulate disk images with `libguestfs` tools
```

Comment:
would be please possible to mention here, that with example pipelines user can try install windows 10 vm from iso file?

Reply:
@ksimon1 should we mention about known issues as well? I saw there is a section here or should we skip as it is a tech preview feature?

Reply:
Since this is a tech preview, I don't think it is necessary